ui/redux/selectors/collections.js

```diff
@@ -40,6 +41,17 @@ export const selectUnpublishedCollectionsList = createSelector(
   (unpublishedCollections) => Object.keys(unpublishedCollections || {})
 );
 
+export const selectCollectionSavedForId = (state: State, id: string) => {
+  const savedIds = selectSavedCollectionIds(state);
+  return savedIds.includes(id);
+};
+
+export const selectSavedCollections = createSelector(
+  selectResolvedCollections,
+  selectSavedCollectionIds,
+  (resolved, savedIds) => Object.fromEntries(Object.entries(resolved).filter(([key, val]) => savedIds.includes(key)))
```

Comment:
I wonder if we still need to use the `fromEntries` polyfill like the rest in this file.
caniuse seems to indicate most browsers support it since 2019, but there must be a reason a polyfill was used (user complaint?)

Reply:
https://odysee-workspace.slack.com/archives/C02PTRGFR70/p1628123205030100